### PR TITLE
meta: Update changelog for release

### PR DESCRIPTION
**Pull request by loewenheim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 ### Various fixes and improvements
 
+- feat(inject): Make sourcemap discovery smarter (#1663) 
+- fix: Add wasm to supported debug-files upload formats (#1683) injection
```
Comment (kamilogorek) [inline CHANGELOG.md:10]:
```suggestion
- fix(debug-files): Add wasm to supported debug-files upload formats (#1683)
```

**Comment by kamilogorek** [comment]
```
fix(inject): Mark paths argument as required (#1662)
fix(inject): Make pragma detection stricter (#1648)
ref(sourcemaps): Do not dedupe artifacts if server wont handle it due to url limit (#1680)
feat(config): Support organization-based tokens (#1673)
feat(issues): Add `issues list` command (#1349)
```

This should also be added. If possible, can you sort feat -> fix -> ref?

**Comment by loewenheim** [comment]
Will do, thanks for the review.

**Comment by loewenheim** [comment]
I added #1688 as well.

**Comment by loewenheim** [comment]
Closing and reopening this because it seems to be buggy.

**Pull request closed without merging**